main.js

```diff
@@ -18,6 +18,16 @@ const serviceMetrics = require('./src/service-metrics');
 const vary = require('./src/middleware/vary');
 const cache = require('./src/middleware/cache');
 const headCssMiddleware = require('./src/middleware/head-css');
+const backendKeys = [];
+if (process.env.FT_NEXT_BACKEND_KEY) {
+	backendKeys.push(process.env.FT_NEXT_BACKEND_KEY);
+}
+if (process.env.FT_NEXT_BACKEND_KEY_OLD) {
+	backendKeys.push(process.env.FT_NEXT_BACKEND_KEY_OLD);
+}
+if (process.env.FT_NEXT_BACKEND_KEY_OLDEST) {
+	backendKeys.push(process.env.FT_NEXT_BACKEND_KEY_OLDEST);
+}
```

Comment:
```js
Object.keys(process.env)
    .filter(key => /FT_NEXT_BACKEND_KEY/.test(key))
    .forEach(key =>  backendKeys.push(process.env[key]));
```

Sorry bit late to the game, but this is a lot cleaner and allows for future addition of keys with same prefix 😄 :